src/components/Fcrackzip/Fcrackzip.tsx

```diff
@@ -1,19 +1,29 @@
-import { Button, LoadingOverlay, Stack, TextInput, NativeSelect, Checkbox } from "@mantine/core";
+import { Button, Stack, TextInput, NativeSelect, Checkbox } from "@mantine/core";
 import { useForm } from "@mantine/form";
 import { useEffect, useCallback, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
-import { UserGuide } from "../UserGuide/UserGuide";
+import { RenderComponent } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "Fcrackzip Tool";
-const description_guide =
-    "Fcrackzip is a tool for cracking password of a protected zip file.\n\n" +
-    "Step 1: Input path of the zip file.\n\n" +
-    "Step 2: Select an Attack Mode (Dictionary or Brute Force)\n\n" +
-    "Step 3: You can save output by checking 'Save Output to File'\n\n" +
-    "Step 4: Click START CRACKING!\n\n";
+// Component title
+const title = "Fcrackzip";
+// Description of the component
+const description_guide = "Fcrackzip is a tool for cracking password of a protected zip file.";
+// Additional props for the RenderComponent
+const steps =
+    "Step 1: Input path of the zip file.\n" +
+    "Step 2: Select an Attack Mode (Dictionary or Brute Force)\n" +
+    "Step 3: You can save output by checking 'Save Output to File'\n" +
+    "Step 4: Click START CRACKING!";
```

Comment:
Make this consistent with the actual button, which is Start Cracking!
```suggestion
    "Step 4: Click Start Cracking!";
```